net/picotls/Portfile

```diff
@@ -59,23 +64,26 @@ patchfiles-append   patch-aligned_alloc.diff
 configure.args-append \
                     -DWITH_AEGIS=OFF \
                     -DWITH_DTRACE=OFF \
+                    -DWITH_FUSION=${with_fusion} \
                     -DWITH_MBEDTLS=OFF
 
-# https://github.com/h2o/picotls/issues/505
+# See net/nss/Portfile for architecture-specific compiler problems
 compiler.blacklist-append \
-                    *gcc-4.* {clang < 500}
-
-# FIXME: fails to build with clang on 10.6–10.7:
-# error: call to undeclared library function 'aligned_alloc'
-# https://github.com/h2o/picotls/issues/514
+                    *gcc-3.* *gcc-4.* {clang < 800}
```

Comment:
Not sure gcc-3.x is a concern anymore, but this can be written shorter as `{*gcc-[34].*}`.